src/MineCase.Protocol/Protocol/Play/ChunkData.cs

```diff
@@ -0,0 +1,114 @@
+﻿using MineCase.Serialization;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace MineCase.Protocol.Play
+{
+    [Packet(0x20)]
+    public sealed class ChunkData : ISerializablePacket
+    {
+        [SerializeAs(DataType.Int)]
+        public int ChunkX;
+
+        [SerializeAs(DataType.Int)]
+        public int ChunkZ;
+
+        [SerializeAs(DataType.Boolean)]
+        public bool GroundUpContinuous;
+
+        [SerializeAs(DataType.VarInt)]
+        public uint PrimaryBitMask;
+
+        [SerializeAs(DataType.VarInt)]
+        public uint Size;
+
+        [SerializeAs(DataType.Array)]
+        public ChunkSection[] Data;
+
+        [SerializeAs(DataType.ByteArray)]
+        public byte[] Biomes;
```

Comment:
@jstzwj Biomes are regions of the terrain in a Minecraft world with varying geographical features.([wiki](http://minecraft.wikia.com/wiki/Biomes))